src/neo4j_graphrag/experimental/components/lexical_graph.py

```diff
@@ -53,8 +55,9 @@ async def run(
         document_info: Optional[DocumentInfo] = None,
     ) -> GraphResult:
         if document_info is None:
-            warnings.warn(
-                "No document metadata provided, the document node won't be created in the lexical graph"
+            logger.info(
+                "No document metadata provided, "
+                "the document node won't be created in the lexical graph"
```

Comment:
I suggest something like

`Document node not created in the lexical graph because no document metadata provided` 

which is a bit more direct I think